usecase/interactor/pokemon.go

```diff
@@ -37,4 +37,14 @@ func (pi *PokemonInteractor) GetPokemon(ID int) (model.Pokemon, error) {
 	}
 
 	return pkm, nil
+}
+
+func(pi *PokemonInteractor) GetPokemonWorker(t string, items int, itemsPerWorker int)([]model.Pokemon){
```

Comment:
Add comment to this method. This should return an error.
```suggestion
func(pi *PokemonInteractor) GetPokemonWorker(t string, items, itemsPerWorker int)([]model.Pokemon){
```